backend/apps/core/middleware/ratelimit.py

```
@@ -0,0 +1,140 @@
+    # Attempt to use raw redis client for ZSET operations (sliding window log).
+    try:
+        redis_url = getattr(settings, "REDIS_URL", None)
+        if redis_url:
+            return redis.from_url(
+                redis_url,
+                socket_connect_timeout=0.2,
+                socket_timeout=0.2,
+                decode_responses=True,
+            )
+    except Exception:
+        pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)